AutoMerge.jl/src/toml.jl

```diff
@@ -1 +1,16 @@
-maybe_parse_toml(f::AbstractString) = ispath(f) ? Pkg.TOML.parsefile(f) : Dict{String, Any}()
+const _toml_cache = Dict{String, Any}()
+
+# Cache parsed toml files for efficiency.
+# Important: Do not mutate the returned data, as it will corrupt the cache.
```

Comment:
This is a downside. Alternatives are not to cache or to return a deepcopy of the cached value.